### ISSUE TITLE
Web UI: With “Show source”, don’t show “Message filtering” button if no messages

### ISSUE THREAD

**Issue by sideshowbarker** [validator/validator#436]

**Comment by j9t** [comment]
Just checked on this—this seems to be the case on https://validator.w3.org/nu/?doc=https%3A%2F%2Fmeiert.com%2F (no errors, no message filtering button).

I tried to test on https://validator.nu/?doc=https%3A%2F%2Fmeiert.com but get errors for the same page, likely because validator.nu runs an older vnu version (22.3.8 vs. 25.12.7).

@sideshowbarker, could it be that this has already been addressed?

**Comment by sideshowbarker** [comment]
Yeah, the problem only occurs if you also choose the option to show the source: 

https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fmeiert.com%2F

**Comment by j9t** [comment]
Ah, overread this. Will take a look!
